src/plugins/auto_batch/src/compiled_model.hpp

```diff
@@ -5,79 +5,75 @@
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 #pragma once
 
-#include <map>
+#include <condition_variable>
```

Comment:
Supposed stl header file put after project header file?

Reply:
this is the code format result